arbitrum-docs/node-running/gentle-introduction-run-node.md

```diff
@@ -6,7 +6,7 @@ author: mahsamoosavi
 
 In order to be able to _interact with_ or _build applications on_ any of the Arbitrum chains, you need access to the corresponding Arbitrum node. Options are:
 
-- You can use third party node providers (see the list [here](/build-decentralized-apps/reference/node-providers)) to get RPC access to fully-managed nodes
+- You can use third party node providers (see the list [here](/build-decentralized-apps/reference/01-node-providers.md)) to get RPC access to fully-managed nodes
```

Comment:
Nitpick: "see the list [here]" > see the [Node Providers List], or something to that effect.